src/config/detekt/detekt.yml

```diff
@@ -22,4 +22,4 @@ style:
   MandatoryBracesLoops:
     active: true
   WildcardImport:
-    active: false
+    active: false
```

Comment:
Please remove this change from this PR - it's not relevant (and files should with a newline anyway)